src/main/target/common_pre.h

```diff
@@ -55,6 +55,7 @@
 #define USE_DSHOT_TELEMETRY
 #define USE_DSHOT_TELEMETRY_STATS
 #define USE_RPM_FILTER
+#define USE_INTERPOLATED_SP
```

Comment:
This should not be part of `STM...` sections in here - these are meant for features that require hardware specific drivers. (Also, this way it is it is bound to be missing from any new MCU types like it is now for H7.)
Instead it should go into the appropriate flash size based section below, which probably is `#if (FLASH_SIZE > 256)` (translating to F4 and better).